### PR TITLE
Feature/ussd nurse rapidpro

### PR DESCRIPTION
**Pull request by OMosimege**
Added a state_facname, added details in state facade that lead to state_facname

Added fixtures for WhatsApp and jembi, and added tests for state_facade that lead to state_facname or loop back to state_faccode

### CODE REVIEW

File: test/ussd_nurse_rapidpro.test.js [hunk 1]
```diff
@@ -17,8 +19,20 @@ describe('app', function() {
                 .setup.config.app({
                     name: 'ussd_nurse_rapidpro',
                     env: 'test',
+                    testing_today: 'July 1, 2018 07:07:07',
+                    testing_message_id: '0170b7bb-978e-4b8a-35d2-662af5b6daee',
```
Comment (erikh360) [inline test/ussd_nurse_rapidpro.test.js]: I don't see where this is used?

File: test/ussd_nurse_rapidpro.test.js [hunk 2]
```diff
@@ -439,7 +454,90 @@ describe('app', function() {
...
+                            .inputs(
+                                {session_event: 'new'}
+                                ,'1' //chooses to start nurse connect
+                                ,'1'  // chooses to sign up
+                                ,'1' //chooses yes to subscribe
+                                ,'0820001004'  // state_msisdn
+                                ,'1' //chooses yes to opt in
+                                ,'12345' //enters facility code
+                            )
+                            .run();
```
Comment (erikh360) [inline test/ussd_nurse_rapidpro.test.js]: We're not checking here if it actually goes to the `state_faccname` state?

**Review by erikh360 — COMMENTED** [review]
Just a few comments.

File: test/ussd_nurse_rapidpro.test.js (hunk 2 above)
Comment (OMosimege) [reply test/ussd_nurse_rapidpro.test.js]: Thanks for pointing this out, need to add more to make sure it goes to the correct state

File: test/ussd_nurse_rapidpro.test.js (hunk 1 above)
Comment (OMosimege) [reply test/ussd_nurse_rapidpro.test.js]: hmm, true. Don't need it

Comment (rudigiesler) [reply test/ussd_nurse_rapidpro.test.js]: Does this still need to be removed?

**Review by rudigiesler — COMMENTED** [review]
Just one small comment. Looks good otherwise

File: test/ussd_nurse_rapidpro.test.js (hunk 1 above)
Comment (OMosimege) [reply test/ussd_nurse_rapidpro.test.js]: Thanks, forgot to remove it

**Review by rudigiesler — APPROVED** [review]

**Pull request merged**